toolchain/docs/adding_features.md

```diff
@@ -290,6 +290,11 @@ updated. The updates needed, can depend on whether the instruction produces a
 type. Look to the comments on those functions for instructions on what is
 needed.
 
+If the instruction owns references other instructions, add a case to
```

Comment:
What does "owns references" mean? Do you mean if an instruction has an InstBlockId?